components/cam/src/physics/cam/physpkg.F90

```diff
@@ -39,8 +39,13 @@ module physpkg
   use cam_logfile,     only: iulog
   use camsrfexch,      only: cam_export
 
-  use modal_aero_calcsize,    only: modal_aero_calcsize_init, modal_aero_calcsize_diag, modal_aero_calcsize_reg
-  use modal_aero_wateruptake, only: modal_aero_wateruptake_init, modal_aero_wateruptake_dr, modal_aero_wateruptake_reg
+  use modal_aero_calcsize,    only: modal_aero_calcsize_init, &
+                                    modal_aero_calcsize_diag, &
```

Comment:
I don't think we need `calcsize_diag` and `calcsize_sub` here as `calcsize_diag` has been moved and calcsize_sub is never called from physpkg.F90

Reply:
good catch, I'll fix this.

Reply:
FYI, we actually still need this for the MMF when running with ECPP, but I should probably move it down into the ECPP driver. I'll leave that for when I get deeper into the ECPP refactor.